src/styles/main.scss

```diff
@@ -1,3 +1,32 @@
+@import "./variables.scss";
+@import "./header.scss";
+@import "./catalog.scss";
+@import "./card.scss";
+@import "./stars.scss";
+@import "./adaptive.scss";
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: $main-color;
+}
+
+div,
+p,
+h2,
+img,
+ul {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Let's create a separate file "normalize" where you can place all reset or default styles that you need.